lib/std/math/big/int_test.zig

```diff
@@ -2123,6 +2124,31 @@ test "big.int bitNotWrap signed multi" {
     try testing.expect((try a.to(SignedDoubleLimb)) == -1);
 }
 
+test "big.int bitNotWrap more than two limbs" {
+    // This test requires int sizes greater than 128 bits.
```

Comment:
I guess creating an issue for llvm as backend not supporting >128bit types + skipping the test for target=wasm + adding a `// TODO` with the link to the issue should be fine. Without proper lowering those types are [not supported by wasm](https://webassembly.github.io/spec/core/syntax/types.html).

Or do you have an idea @Luukdegram?

Reply:
Thanks to your help reproducing this locally, I understand the problem now. I was mistaking this as not working with the wasm32 backend, but it's the LLVM backend, triggered when testing with `-target wasm32-wasi`. Exactly as you said! I was mixing up backend with target.

So for now I can make a TODO as suggested, although there might not be much left to actually test here unless I change the code not to use >128-bit ints. Alternatively I could delete this test, as the bug is indirectly covered by the popcount test.